src/locales/en.json

```diff
@@ -617,7 +617,8 @@
     "title": "Audio Category",
     "music": "Music",
     "sound-effects": "Sound Effects",
-    "podcast": "Podcast"
+    "podcast": "Podcast",
+    "undefined": "Undefined category"
```

Comment:
@panchovm Can we omit the category if we don't know instead of showing "Undefined category"?

Reply:
Yes. Definitely.

Reply:
Deleting then... 🧹  Thanks!